app/controllers/presentations_controller.rb

```diff
@@ -20,6 +29,26 @@ def create
     end
   end
 
+  def update
+    @presentation = Presentation.find(params[:id])
+    authorize @presentation
+
+    if @presentation.update(presentation_params)
+      flash[:notice] = 'Presentation was successfully updated.'
+      redirect_to @presentation
+    else
+      render 'edit'
+    end
+  end
+
+  def destroy
+    @presentation = Presentation.find(params[:id])
+    authorize @presentation
+    @presentation.destroy
```

Comment:
As Brendan mentioned, this should be `destroy!` since the return value is not used